<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-SKPPPFG" ignored="0" name="AdhocAuditManagementRejection1" tests="10" failures="2" timestamp="2021-08-25T19:26:35 IST" time="666.632" errors="0">
  <testcase name="@AfterMethod testIT" time="-1.629899795529E9" classname="com.pss.qms.AuditManagementAdhocRejection.AuditeeResponseApproval">
    <failure type="java.lang.RuntimeException" message="Test skipped due to a detected failure in the overall suite.">
      <![CDATA[java.lang.RuntimeException: Test skipped due to a detected failure in the overall suite.
at com.pss.qms.Listners.SuiteFailingInvokedMethodListener.beforeInvocation(SuiteFailingInvokedMethodListener.java:17)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod testIT -->
  <testcase name="@AfterClass tearDown" time="-1.629899795532E9" classname="com.pss.qms.login.AMLoginDetails">
    <failure type="java.lang.RuntimeException" message="Test skipped due to a detected failure in the overall suite.">
      <![CDATA[java.lang.RuntimeException: Test skipped due to a detected failure in the overall suite.
at com.pss.qms.Listners.SuiteFailingInvokedMethodListener.beforeInvocation(SuiteFailingInvokedMethodListener.java:17)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="@BeforeClass setUp" time="-1.629899795534E9" classname="com.pss.qms.login.AMLoginDetails">
    <failure type="java.lang.RuntimeException" message="Test skipped due to a detected failure in the overall suite.">
      <![CDATA[java.lang.RuntimeException: Test skipped due to a detected failure in the overall suite.
at com.pss.qms.Listners.SuiteFailingInvokedMethodListener.beforeInvocation(SuiteFailingInvokedMethodListener.java:17)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass setUp -->
  <testcase name="@AfterClass tearDown" time="-1.62989979567E9" classname="com.pss.qms.login.AMLoginDetails">
    <failure type="java.lang.RuntimeException" message="Test skipped due to a detected failure in the overall suite.">
      <![CDATA[java.lang.RuntimeException: Test skipped due to a detected failure in the overall suite.
at com.pss.qms.Listners.SuiteFailingInvokedMethodListener.beforeInvocation(SuiteFailingInvokedMethodListener.java:17)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="toAuditObservationsRejectedApprove" time="58.392" classname="com.pss.qms.AuditManagementAdhocRejection.AuditObservationsRejectedApprove"/>
  <testcase name="toAuditObservationsReview" time="53.365" classname="com.pss.qms.AuditManagementAdhocRejection.AuditObservationsReview"/>
  <testcase name="AuditObservationApprove" time="59.927" classname="com.pss.qms.AuditManagementAdhocRejection.AuditObservationsApprove"/>
  <testcase name="AuditeeResponse" time="170.238" classname="com.pss.qms.AuditManagementAdhocRejection.AuditeeResponse"/>
  <testcase name="toAuditeeResponseReviewReject" time="60.236" classname="com.pss.qms.AuditManagementAdhocRejection.AuditeeResponseReviewReject"/>
  <testcase name="toAuditeeResponseRejectedApprove" time="66.282" classname="com.pss.qms.AuditManagementAdhocRejection.AuditeeResponseRejectedApprove"/>
  <testcase name="toAuditeeResponseReview" time="61.386" classname="com.pss.qms.AuditManagementAdhocRejection.AuditeeResponseReview"/>
  <testcase name="toAuditeeResponseApproveReject" time="63.256" classname="com.pss.qms.AuditManagementAdhocRejection.AuditeeResponseApproveReject"/>
  <testcase name="AuditeeResponseApprove" time="16.185" classname="com.pss.qms.AuditManagementAdhocRejection.AuditeeResponseApproval">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at com.pss.qms.AuditManagementAdhocRejection.AuditeeResponseApproval.methodToDoAuditeeResponseApprove(AuditeeResponseApproval.java:170)
at com.pss.qms.AuditManagementAdhocRejection.AuditeeResponseApproval.AuditeeResponseApprove(AuditeeResponseApproval.java:88)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- AuditeeResponseApprove -->
  <testcase name="toCloseAudit" time="0.003" classname="com.pss.qms.AuditManagementAdhocRejection.CloseAudit">
    <failure type="java.lang.RuntimeException" message="Test skipped due to a detected failure in the overall suite.">
      <![CDATA[java.lang.RuntimeException: Test skipped due to a detected failure in the overall suite.
at com.pss.qms.Listners.SuiteFailingInvokedMethodListener.beforeInvocation(SuiteFailingInvokedMethodListener.java:17)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- toCloseAudit -->
</testsuite> <!-- AdhocAuditManagementRejection1 -->
